module/rds.py

```diff
@@ -136,165 +178,1292 @@
       default: null
     db_description:
       description:
-        - Description of a database to create within the instance.  If not specified then no database is created.
+        - Description of a database to create within the instance. 
       required: false
       default: null
     character_set_name:
       description:
         - Associate the DB instance with a specified character set.
       required: false
       default: null
-    username:
+    maint_window:
       description:
-        - Master database username. Used only when command=create.
+        - "Maintenance window in format of ddd:hh24:mi-ddd:hh24:mi.  (Example: Mon:22:00-Mon:23:15) If not specified
+        then a random maintenance window is assigned."
       required: false
       default: null
-    password:
+    preferred_backup_time:
       description:
-        - Password for the master database username. Used only when command=create or command=modify.
+        - Backup time, in the format ofHH:mmZ- HH:mm Z.This parameter is required if preferred_backup_period and
+         backup_retention_period is passed.
       required: false
-      default: null
-    maint_window:
+      default: null      
+    preferred_backup_period:
       description:
-        - "Maintenance window in format of ddd:hh24:mi-ddd:hh24:mi.  (Example: Mon:22:00-Mon:23:15) If not specified then a random maintenance window is assigned."
+        - Backup period.
       required: false
       default: null
-    backup_window:
+      aliases: ['backup_window']
+    backup_retention_period:
       description:
-        - Backup window in format of hh24:mi-hh24:mi.  If not specified then a random backup window is assigned.
+        - "Number of days backups are retained.  Set to 0 to disable backups.  Default is 7 day.  Valid range: 7-730.
       required: false
       default: null
-    backup_retention:
+      aliases: ['backup_retention']
+    db_tags:
       description:
-        - "Number of days backups are retained.  Set to 0 to disable backups.  Default is 1 day.  Valid range: 0-35.
+        - A hash/dictionaries of db tags, '{"tag_key": "tag_value"}', tag_key must be not null when tag_value isn't null.
       required: false
       default: null
+    wait:
+      description:
+        - wait for the RDS instance to be in state 'running' before returning.
+      required: false
+      default: "no"
+      choices: [ "yes", "no" ]
+    wait_timeout:
+      description:
+        - how long before wait gives up, in seconds
+      required: false
+      default: 300
 
+function: change rds instance type
+  description: change rds instance type
+  command: modify
+  options:
+    instance_id:
+      description:
+        - ID of the database to change.
+      required: true
+      default: null      
+    region:

```

Comment:
In the line 239, please unify the parameter region with above 'alicloud_region'.